examples/secure-third-party-request/.secret-example.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
If I understand well, this json file is a .env file for developers to define the secrets values themselves and be able to work locally?

Reply:
yes exactly. Sadly we cannot use .env files as vite doesnt expose then with their local-sever